ntoskrnl/ke/i386/ctxswitch.S

```diff
@@ -61,12 +61,12 @@ PUBLIC @KiSwapContextInternal@0
  * BOOLEAN FASTCALL
  * KiSwapContext(PKTHREAD CurrentThread, PKTHREAD TargetThread);
  *
+ * \param WaitIrql<cl>
```

Comment:
"<cl>" ??

Reply:
!!

Reply:
hm, now that I think again about it, isn't it the register name?